src/main/java/org/sonarlint/intellij/fix/ShowFixSuggestion.kt

```diff
@@ -37,25 +36,25 @@ import org.sonarlint.intellij.ui.inlay.InlayManager
 import org.sonarlint.intellij.util.getDocument
 import org.sonarsource.sonarlint.core.rpc.protocol.client.fix.FixSuggestionDto
 
-class ShowFixSuggestion(private val project: Project, private val file: VirtualFile, private val fixSuggestion: FixSuggestionDto) {
+class ShowFixSuggestion(
+    private val project: Project, private val file: VirtualFile, private val fixSuggestion: FixSuggestionDto
+) {
 
     fun show() {
-        val fileEditorManager = FileEditorManager.getInstance(project)
-        val psiFile = PsiManager.getInstance(project).findFile(file) ?: return
```

Comment:
In the documentation, it is mentioned that psi file operations should not be done on the UI thread. Why are we moving these back to the UI thread again?

Reply:
`PsiManager.getInstance(project).findFile(file)` requires to have read access

Reply:
Yes but we don't need UI Thread access

Reply:
Right, can move to read access only